jf_agent/main.py

```diff
@@ -605,7 +372,16 @@ def upload_file(filename, path_to_obj, signed_url):
 
     print('Sending data to Jellyfish... ')
 
-    signed_urls = get_signed_url(os.listdir(config.outdir))
+    # obtain file names from the directory
+    _, directories, filenames = next(os.walk(config.outdir))
+
+    for directory in directories:
+        path = os.path.join(config.outdir, directory)
+        for file_name in os.listdir(path):
+            print(f'{directory}/{file_name}')
```

Comment:
remove this line

Reply:
What does this mean?

Reply:
I will add a comment to clarify this code. Since the code creates subdirectories, I want to make sure that all those paths are included in `get_signed_urls`, otherwise we will not include them in their upload to s3.